doc/Building.md

```diff
@@ -11,21 +11,26 @@
     cd vscode
     git checkout <git tag or branch>
 5. Install project dependencies
-    yarn
-    
+    npm install
+6. Get back to the repository root folder
+    cd ..
+
 ## Building
-Building the exensions from VS Code is done simply with 
+
+Building the exensions from VS Code is done simply from this repository root folder with
 
     yarn build:extensions
```

Comment:
Yarn?

Reply:
no, the vscode-builtin-extension project itself still relies on yarn. Only vscode submodule was migrated to npm. I was waiting for the success on theia project migration to check if it should move also to npm or any other choice. By the time of the PR, this was not yet finalized. 
I should have mentionned it in the review text. BTW, should we use the default PR template for this project also? The follow up section would make sense here, as the project may be also upgrade once npm migration is finalized on main theia repo.